src/test/java/org/broadinstitute/hellbender/tools/GenomicsDBImportIntegrationTest.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.hellbender.tools;
+
+import com.intel.genomicsdb.GenomicsDBFeatureReader;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.CloseableTribbleIterator;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.tribble.readers.PositionalBufferedStream;
+import htsjdk.variant.bcf2.BCF2Codec;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFCodec;
+import org.apache.commons.io.FileUtils;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.GenomicsDBTestUtils;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+public final class GenomicsDBImportIntegrationTest extends CommandLineProgramTest {
+
+  private static final String TEST_OUTPUT_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/tools";
+  private static final File GENOMICSDB_WORKSPACE = new File(TEST_OUTPUT_DIRECTORY + "/tiledb-ws");
+  private static final String GENOMICSDB_ARRAYNAME = "gatk4-genomicsdb-test-0";
+  private static final File TEST_CALLSETMAP_JSON_FILE = new File(TEST_OUTPUT_DIRECTORY + "/callset.json");
+  private static final File TEST_VIDMAP_JSON_FILE = new File(TEST_OUTPUT_DIRECTORY + "/vidmap.json");
+
+  private static final String hg00096 = publicTestDir + "large/gvcfs/HG00096.g.vcf.gz";
+  private static final String hg00268 = publicTestDir + "large/gvcfs/HG00268.g.vcf.gz";
+  private static final String na19625 = publicTestDir + "large/gvcfs/NA19625.g.vcf.gz";
+  private static final String combined = publicTestDir + "large/gvcfs/combined.gatk3.7.g.vcf.gz";
+
+  private static final File TEST_REFERENCE_GENOME = new File(publicTestDir + "/large/Homo_sapiens_assembly38.20.21.fasta");
+
+  @Override
+  public String getTestedClassName() {
+    return GenomicsDBImport.class.getSimpleName();
+  }
+
+  @Test
+  public void testGenomicsDBImporter() throws IOException {
+
+    final ArgumentsBuilder args = new ArgumentsBuilder();
+    args.add("-GW"); args.add(GENOMICSDB_WORKSPACE.getAbsolutePath());
+    args.add("-GCWS"); args.add(true);
+    args.add("-GA"); args.add(GENOMICSDB_ARRAYNAME);
+    args.add("-R"); args.add(TEST_REFERENCE_GENOME);
```

Comment:
Getting an exception saying GATKTool requires a reference genome.

org.broadinstitute.hellbender.exceptions.UserException: We currently require a sequence dictionary (from a reference, a source of reads, or a source of variants) to process intervals. This restriction may be removed in the future.

	at org.broadinstitute.hellbender.engine.GATKTool.initializeIntervals(GATKTool.java:270)
	at org.broadinstitute.hellbender.engine.GATKTool.onStartup(GATKTool.java:434)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.runTool(CommandLineProgram.java:114)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMainPostParseArgs(CommandLineProgram.java:179)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:198)
	at org.broadinstitute.hellbender.Main.runCommandLineProgram(Main.java:121)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:108)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:128)
	at org.broadinstitute.hellbender.CommandLineProgramTest.runCommandLine(CommandLineProgramTest.java:28)
	at org.broadinstitute.hellbender.utils.test.CommandLineProgramTester.runCommandLine(CommandLineProgramTester.java:92)